flink-kubernetes-standalone/src/main/java/org/apache/flink/kubernetes/operator/kubeclient/Fabric8FlinkStandaloneKubeClient.java

```diff
@@ -57,14 +58,14 @@ public void stopAndCleanupCluster(String clusterId) {
                 .apps()
                 .deployments()
                 .withName(StandaloneKubernetesUtils.getJobManagerDeploymentName(clusterId))
-                .cascading(true)
+                .withPropagationPolicy(DeletionPropagation.ORPHAN)
```

Comment:
@usamj could you please confirm that this was the intention here.

Reply:
Why note foreground? Orphan would not delete pods, configmaps etc right?

Reply:
That makes more sense, to me - the javadoc was ambiguous tbh. I prefer foreground unless @usamj or @dannycranmer disagrees.

Reply:
I agree, foreground is more in-line with the behaviour that we want.